task10.rb

```diff
@@ -0,0 +1,32 @@
+# Return the maximum product of three numbers in an array, so [-8, -9, 1, 2, 7] outputs 504 (which is -8*-9*7)
+
+def task10(arr)
+  raise ArgumentError, "Not a valid format. Need an array" if arr.class != Array
+  raise ArgumentError, "Array is empty. This is not valid" if arr.empty?
+  raise ArgumentError, "Not a valid format. Need an array with more then two integer or float numbers inside" if arr.length < 3 
+  arr.each do |number|
+    raise ArgumentError, "Not a valid format. Need an array with more then two integer or float numbers inside" if number.class != Integer && number.class != Float
```

Comment:
@MaraEfimova I believe this is not the same error as the one above, if it is - then combine the conditionals and do not repeat it :wink: